scripts/makediff.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+base_file=$1
+
+merge_base=$(git merge-base HEAD origin/master)
+
+latexdiff --version
+
+echo <<EOF > config 
```

Comment:
This does nothing because `echo` should be `cat`.

If that were fixed, until a few days ago it would still do nothing due to ftilmann/latexdiff#160. I guess avoiding this bug is why your Docker image pins a particular latexdiff commit, though there is no comment or commit message explaining that. (As it happens, latexdiff 1.3.0 has been released in the last few days with this bug fixed.)

If both of those were fixed, this still does nothing because PICTUREENV takes effect within latexdiff's `sub preprocess` **after** when you would need it to.

So this _config_ stuff just plain doesn't work and you are stuck with `--graphics-markup none`.